modules/@apostrophecms/notification/index.js

```diff
@@ -160,6 +160,35 @@ module.exports = {
       return self.db.deleteMany({ _id });
     }
   }),
+  apiRoutes(self) {
+    return {
+      post: {
+        'event-clear': async function (req) {
+          if (!req.body.id) {
```

Comment:
Also, multiword route names in the piece type module are written in natural English order rather than "noun then verb". For example:

```
':_id/revert-draft-to-published': async (req) => { ... }
```

So I would recommend:

```
`:_id/clear-event`: async req => { ... }
```